docs/processors/standalone/index.mdx

```diff
@@ -4,9 +4,89 @@ sidebar_position: 3
 
 # Standalone processors
 
-:::warning
+These are processors you'd write yourself in cases the [Built-in](/docs/processors/builtin/) ones don't meet your needs. 
 
-🚧 Page is under construction, content will be added soon.
-This page should include general info about how to use standalone processors.
+## How to write one
 
-:::
\ No newline at end of file
+Thanks to our [Web Assembly (Wasm) processor](/docs/processors/standalone/how-it-works) you can start writing processors in any language that can be compiled to Web Assembly. As a start, Conduit already provides a [conduit-processor-sdk](https://github.com/ConduitIO/conduit-processor-sdk) that will let you [write a processor](/docs/processors/standalone/building) in Golang.
```

Comment:
Nitpick: it looks like Go is the official language name.

Reply:
Hmm, maybe we're not talking about the same thing. I was suggesting replacing `in Golang` with `in Go`.

Reply:
@hariso oh! I see! I'll change